### PR TITLE
 Add add_simple_tax_schema_entry functionality

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/115581

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Ready for review

File: src/Tribe/Repository.php [hunk 1]
```diff
@@ -295,6 +298,10 @@
 	 * @var array A map of schema slugs and their meta keys to be queried.
 	 */
 	protected $simple_meta_schema = array();
+	/**
```
Comment (lucatume) [inline src/Tribe/Repository.php]: Add a blank line before.

File: src/Tribe/Repository.php [hunk 2]
```diff
@@ -295,6 +298,10 @@
...
+	 * @var array A map of schema slugs and their taxonomies to be queried.
+	 */
+	protected $simple_tax_schema = array();
 	/**
```
Comment (lucatume) [inline src/Tribe/Repository.php]: add a blank line before

Comment (sc0ttkclark) [reply src/Tribe/Repository.php]: Sorry, was following the formatting that was there, lots of things without blank lines in here

**Pull request merged**